src/main/java/hudson/plugins/ansicolor/AnsiHtmlOutputStream.java

```diff
@@ -473,6 +476,7 @@ private String getRgbColor(int r, int g, int b) throws IOException {
         return "#" + String.format("%02X", r) + String.format("%02X", g) + String.format("%02X", b);
     }
 
+    @SuppressFBWarnings("UC_USELESS_CONDITION")
```

Comment:
I think you are right on where this is happening - the `@SuppressFBWarnings` can't be applied to only that section unfortunately. 

I made a new commit that I'll push up shortly that hopefully clarifies this method and removes the suppression.